src/components/MainTitle.vue

```diff
@@ -26,22 +37,32 @@ export default {
     font-family: $ff-secondary;
     font-size: $fs-large-alt;
     line-height: 49px;
+    margin-top: 50px;
     padding: 35px 0;
     text-align: center;
 
     & span {
       color: $cl-grey;
       display: block;
     }
+
+    &--left{
+       text-align: left;
```

Comment:
I see that you set `text-align: left` 3 times in this file and `text-align: center` once. Remember that `left` is a default value. That means you first overwrite it, then you overwrite it again 3 times. You could optimize it by finding a single condition, where it should be center and apply `center` only if that happens. This will remove some lines of code. You may need to invert the media query, for example, you apply styles for `max-width` now, but you can create a media query for `min-width` condition if it gives you less lines of code. 😃